### ISSUE TITLE
Copying on windows does not copy

### ISSUE THREAD

**Issue by TurnItOff-OnAgain** [flameshot-org/flameshot#2487]
### Flameshot Version

11.0.0 (ad1bf28)

### Installation Type

Using the ready-made package from Github Releases

### Operating System type and version

Windows 10 21H1

### Description

After taking a screenshot I either do ctrl+c or double left click to copy and it does not copy. Initially was an upgrade install from a previous version. I uninstalled and did a fresh install and I still cannot copy. Install was done via the precompiled MSI.

### Steps to reproduce

Take a screenshot
ctrl+c or left double click to copy

### Screenshots or screen recordings

Video recording seems to be funky. Here is a link to it as well
https://streamable.com/spd0qx

https://user-images.githubusercontent.com/26926284/158206250-be630606-e9e5-4935-91b8-fe220ea63145.mp4

### System Information

Flameshot v11.0.0 (ad1bf28)
Compiled with Qt 5.15.2
winnt: 10.0.19043
windows: 10

![image](https://user-images.githubusercontent.com/26926284/158206498-9fbec47c-440d-4285-a132-c56912c93c00.png)

**Comment by TurnItOff-OnAgain** [comment]
Uninstalled V11 and went back to 10.2 (2bf8a92) and copying is working correctly.

**Comment by mmahmoudian** [comment]
Please provide your Flameshot config

**Comment by louix** [comment]
I experience the same problem, only when this setting was set:
```ini
useJpgForClipboard=true
```
corresponds to the "Use JPG format for clipboard" UI option.

**Comment by DennisDyallo** [comment]
Same problem here on Win 10 fresh install. First time Flameshot user. Tried the v10 and its working perfectly. Thanks for great software hopefully we can get it working on the v11 again soon

**Comment by thielfab** [comment]
> I experience the same problem, only when this setting was set:
> 
> ```ini
> useJpgForClipboard=true
> ```
> 
> corresponds to the "Use JPG format for clipboard" UI option.

I can confirm this. If the JPG option is not set, the clipboard works fine! 
Running Flameshot v11.0.0 (ad1bf28) on W10 21H2.

**Comment by SleepDaemon** [comment]
> > I experience the same problem, only when this setting was set:
> > ```ini
> > useJpgForClipboard=true
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > corresponds to the "Use JPG format for clipboard" UI option.
> 
> I can confirm this. If the JPG option is not set, the clipboard works fine! Running Flameshot v11.0.0 ([ad1bf28](https://github.com/flameshot-org/flameshot/commit/ad1bf288157f7ec3f1a6251ee5c71c751aef4d72)) on W10 21H2.

Same here as well, setting to JPG doesnr work on V12.1.0 but worked fine on V10

**Comment by TurnItOff-OnAgain** [comment]
I got a new laptop and am now on windows 11, flameshot v12.1.0. Having the "Use JPG format for clipboard" option checked 

Config attached. 

[flameshot.txt](https://github.com/flameshot-org/flameshot/files/10194846/flameshot.txt)

**Comment by mmahmoudian** [comment]
Yes, this is a known issue. For now the workaround is to remove that JPEG checkbox

**Comment by SleepDaemon** [comment]
> Yes, this is a known issue. For now the workaround is to remove that JPEG checkbox

I am aware but PNG has a large file size for simple screenshots. It makes more sense to use JPEG or JPG (idk if they are the samething let me know if I am wrong) as their size is way less and far more suitable in this use case. But till this get fixed I will just keep it unchecked.

**Comment by wyndon** [comment]
This is a limitation of Windows, it quite literally doesn't support JPG images in clipboard. It only supports PNG or bitmap. This setting should just be greyed out in the Windows builds with a tooltip that explains why it's not supported.

**Comment by ElTh0r0** [comment]
> This is a limitation of Windows, it quite literally doesn't support JPG images in clipboard

I tried a short internet research, but I couldn't find a 100% clear statement, but only 'vague hints'. Anyway I think you are right!

I'd propose to _force_ `useJpgForClipboard=false` on Windows even if users may currently have set it to `true`. Do you agree? If so, I can prepare a PR for this change.
Do you think it is better to disable (grey out) the option or even hide it completely for Windows users?
